magstock/templates/static_views/stafferComps.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
MAGStock volunteering is so much different than regular MAGFest volunteering that it made sense to just override the entire template.